cobigen-eclipse/cobigen-eclipse-test/src/main/java/com/devonfw/cobigen/eclipse/test/HealthCheckTest.java

```diff
@@ -46,6 +46,8 @@ public void testHealthCheckIfTemplateProjecNotCopiedIntoWS() throws Exception {
     EclipseUtils.importExistingGeneralProject(bot, tmpFolder.getAbsolutePath(), false);
     EclipseUtils.updateMavenProject(bot, ResourceConstants.CONFIG_PROJECT_NAME);
 
-    EclipseCobiGenUtils.runAndCaptureHealthCheck(bot);
+    EclipseCobiGenUtils.runAndCaptureHealthCheckWithMonolithicConfiguration(bot);
   }
+
+  // TODO: Add a health check test for new template sets, see: https://github.com/devonfw/cobigen/issues/1648
```

Comment:
Create an issue and reference it or remove the TODO

Reply:
done.